### PR TITLE
Consolidate fake WCS generation logic

### PR DESCRIPTION
**Pull request by jeremykubica**
Clean up the testing code by consolidating the specification of fake WCSs into a single location and calling that.  Also expands some of the tests for WCS generation to make sure they are reasonable as fake data.

### CODE REVIEW

File: tests/test_wcs_utils.py [hunk 1]
```diff
@@ -45,16 +45,54 @@ def test_wcs_to_dict(self):
...
+            pri = fits.PrimaryHDU()
+            self.assertFalse("CRVAL1" in pri.header)
+            self.assertFalse("CRVAL2" in pri.header)
+            self.assertFalse("CRPIX1" in pri.header)
+            self.assertFalse("CRPIX2" in pri.header)
+
+            append_wcs_to_hdu_header(wcs_info, pri.header)
+            for key in self.header_dict:
+                self.assertTrue(key in pri.header)
+                self.assertAlmostEqual(pri.header[key], self.header_dict[key])
```
Comment (wilsonbb) [inline tests/test_wcs_utils.py:65]: I'm a bit confused here. I presume the subTests are supposed to be within scope of the the `for use_dictionary in [True, False]` loop?

File: tests/test_wcs_utils.py [hunk 2]
```diff
@@ -45,16 +45,54 @@ def test_wcs_to_dict(self):
...
+        self.assertEqual(wcs_dict["CRPIX1"], 100.0)
+        self.assertEqual(wcs_dict["CRPIX2"], 50.0)
+        self.assertEqual(wcs_dict["CDELT1"], 0.01)
+        self.assertEqual(wcs_dict["CDELT2"], 0.01)
+        self.assertEqual(wcs_dict["CTYPE1A"], "LINEAR  ")
+        self.assertEqual(wcs_dict["CTYPE2A"], "LINEAR  ")
+        self.assertEqual(wcs_dict["CUNIT1A"], "PIXEL   ")
+        self.assertEqual(wcs_dict["CUNIT2A"], "PIXEL   ")
+
+        # Test the we can convert to a WCS and extra predictions.
```
Comment (wilsonbb) [inline tests/test_wcs_utils.py:84]: Nit: s/Test the we/Test that we

File: tests/test_wcs_utils.py (hunk 1 above)
Comment (jeremykubica) [reply tests/test_wcs_utils.py]: Good catch. It's off that this worked during testing.

Comment (jeremykubica) [reply tests/test_wcs_utils.py]: I fixed this.

**Review by wilsonbb — APPROVED** [review]

**Pull request merged**